docs/content/integrations/duckdb/using-duckdb-with-dagster.mdx

```diff
@@ -130,19 +292,15 @@ from dagster import asset
 
 
 @asset
-def iris_cleaned(iris_dataset: pd.DataFrame) -> pd.DataFrame:
-    return iris_dataset.dropna().drop_duplicates()
+def iris_setosa(iris_dataset: pd.DataFrame) -> pd.DataFrame:
+    return iris_dataset[iris_dataset["species"] == "Iris-setosa"]
 ```
 
-In this example, we want to provide the `iris_dataset` asset from the [Store a Dagster asset as a table in DuckDB](#store-a-dagster-asset-as-a-table-in-duckdb) example to the `iris_cleaned` asset.
-
-In `iris_cleaned`, the `iris_dataset` parameter tells Dagster that the value for the `iris_dataset` asset should be provided as input to `iris_cleaned`. If this feels too magical for you, refer to the [docs for explicitly specifying dependencies](/concepts/assets/software-defined-assets#defining-explicit-managed-loading-dependencies).
-
-When materializing these assets, Dagster will use the `DuckDBPandasIOManager` to fetch the `IRIS.IRIS_DATASET` as a Pandas DataFrame and pass this DataFrame as the `iris_dataset` parameter to `iris_cleaned`. When `iris_cleaned` returns a Pandas DataFrame, Dagster will use the `DuckDBPandasIOManager` to store the DataFrame as the `IRIS.IRIS_CLEANED` table in DuckDB.
+In this asset, you're providing the `iris_dataset` asset as a dependency to `iris_setosa`. By supplying `iris_dataset` as a parameter to `iris_setosa`, Dagster knows to use the `DuckDBPandasIOManager` to load this asset into memory as a Pandas DataFrame and pass it as an argument to `iris_setosa`. Next, a DataFrame that only contains the data for the _Iris Setosa_ species is created and returned. Then the `DuckDBPandasIOManager` will store the DataFrame as the `IRIS.IRIS_SETOSA` table in DuckDB.
 
 ---
```

Comment:
Sorry I missed this last time - could you remove this?  The rule is only needed above H2 headings